packages/core/contracts/common/implementation/DSProxyFactory.sol

```diff
@@ -0,0 +1,203 @@
+// This file was originally take from from Maker: DS Proxy Factory and modified. The original source code can be found
```

Comment:
Similar to uniswap, are there packages exposed that include the ABIs and bytecode for this factory?

Reply:
not that I can find but that is a good point! I don think this is too bloated though, wdyt?